README.md

```diff
@@ -19,6 +19,7 @@ Yet Another GitHub Action to notify slack.
 | text_on_success   | any string                                                 | ''                    | `text` field on success                   |
 | text_on_fail      | any string                                                 | ''                    | `text` field on failure                   |
 | text_on_cancel    | any string                                                 | ''                    | `text` field on cancellation              |
+| text_on_skipped   | any string                                                 | ''                    | `text` field on skipped                   |
```

Comment:
@shutosekiguchi 

> | status            | 'success' or 'failure' or 'cancelled' or 'custom'          | ''                    | Use `${{ job.status }}`.                  |

What is `skipped` status? I could not find at https://docs.github.com/ja/actions/learn-github-actions/contexts#job-context.

Reply:
You want to support https://github.com/technote-space/workflow-conclusion-action by any chance?

> status: ${{ env.WORKFLOW_CONCLUSION }} # neutral, success, skipped, cancelled, timed_out, action_required, failure

If so, could you support `neutral` and `timed_out` and `action_required` in addition to `skipped`?
Also, you could write description about supporting workflow-conclusion-action on README.md?

Reply:
You want to support https://github.com/technote-space/workflow-conclusion-action by any chance?

> status: ${{ env.WORKFLOW_CONCLUSION }} # neutral, success, skipped, cancelled, timed_out, action_required, failure

If so, could you support `neutral` and `timed_out` and `action_required` in addition to `skipped`?
Also, you could write description about supporting workflow-conclusion-action on README.md?

Reply:
@shutosekiguchi I recognized that https://github.com/technote-space/workflow-conclusion-action is in public archive. Do you have any alternatives?

Reply:
I see, we are currently considering our operational approach. Thank you.

Reply:
Okay, thanks!